src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/Microsoft.AspNetCore.Razor.LanguageServer.Common.csproj

```diff
@@ -4,6 +4,8 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains common assets that are used in the Razor language server and other assemblies.</Description>
     <EnableApiCheck>false</EnableApiCheck>
+    <!-- Suppress warnings about depending on non-stable version of Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X -->
+    <NoWarn>$(NoWarn);NU5104</NoWarn>
```

Comment:
Why are we depending on a non-stable version of that package? Sounds like a valid warning.

Of course, if there's an explanation, looks good.

Reply:
That package is built by this repo, it's listed as a dependency in the .nuspec (which is what the error is complaining about). It may be that _that's_ incorrect, but I'm not sure